### ISSUE TITLE
Can OIH records be 'metadata only' 

### ISSUE THREAD

**Issue by dblear** [marco-bolo/dataset-catalogue#34]
Does an OIH metadata record have to resolve to a dataset?
If so, can thar be a portal 'landing page' or a publication (ie PDF) or does it need to be a machine actionable endpoint? (eg OGC API or direct file download)

**Comment by pieterprovoost** [comment]
That seems fine to me, just leave the https://schema.org/distribution property empty.

**Comment by pbuttigieg** [comment]
> Does an OIH metadata record have to resolve to a dataset?


No, but a schema:Dataset JSON-LD record should point to the data it describes via the distribution property 

other types need not point data sets 

> If so, can thar be a portal 'landing page' or a publication (ie PDF) or does it need to be a machine actionable endpoint? (eg OGC API or direct file download)


if it's a landing page or something similar, you can use the url property. But it must point to the dataset and not some document about the dataset (like a paper). One can use the citation property to point to resources that talk about the subject data

**Comment by dblear** [comment]
Slight tangent, so are the records from MEDIN schema:Dataset ? Because there's many, many on there that don't have any value in the distribution property.  Will they be dropped from OIH?

**Comment by pbuttigieg** [comment]
> Slight tangent, so are the records from MEDIN schema:Dataset ? Because there's many, many on there that don't have any value in the distribution property.  Will they be dropped from OIH?

Most are, other Types are fine too

No distribution will not affect their discoverability in ODIS or OIH, but it will make rapid, programmatic download harder.

**Comment by pbuttigieg** [comment]
Let's leave this open until we document this answer in D1.1 and in this repo @dblear

**Comment by pbuttigieg** [comment]
Added note on this in https://github.com/marco-bolo/metadata-catalogue/commit/a989230931b702dbf37c4e81e2941392b190a1de
